app/i18n/de.json

```diff
@@ -309,7 +309,7 @@
         "fiat": "Währung",
         "deposit_address": "Deposit-Adresse",
         "conflict": "Die Route existiert bereits!",
-        "exchange_rate_warning": "Achtung, wir kaufen dieses Asset auf der DEX und können keinen Wechselkurs garantieren! Die Preise auf der DEX können deutlich tiefer oder höher als auf anderen Exchanges sein, was sich auch negativ auf den Wechselkurs auswirken kann.\n\nBitte beachte beim Kauf von dToken Assets, dass wenn du diese zu einem späteren Zeitpunkt wieder in Crypto-Token (DFI, dBTC, dETH...), Stablecoins (dUSDC, dUSDT) tauschen, oder über DFX gegen FIAT (EUR, USD, FR...) verkaufen möchtest, in diesem Zug eine DEX Stabilisierungsgebühr in Höhe von aktuell ca. 29% anfällt. Bei einem späteren Tausch innerhalb der dToken (dUSD, dTSLA, dAMZN, dCOIN …) fällt die vorgenannte DEX Stablisierungsgebühr nicht an.",
+        "exchange_rate_warning": "Achtung, wir kaufen dieses Asset auf der DEX und können keinen Wechselkurs garantieren! Die Preise auf der DEX können deutlich tiefer oder höher als auf anderen Exchanges sein, was sich auch negativ auf den Wechselkurs auswirken kann.\n\nBitte beachte beim Kauf von dToken Assets, dass wenn du diese zu einem späteren Zeitpunkt wieder in Crypto-Token (DFI, dBTC, dETH...), Stablecoins (dUSDC, dUSDT) tauschen, oder über DFX gegen FIAT (EUR, USD, FR...) verkaufen möchtest, in diesem Zug eine DEX Stabilisierungsgebühr in Höhe von aktuell ca. 12% anfällt. Bei einem späteren Tausch innerhalb der dToken (dUSD, dTSLA, dAMZN, dCOIN …) fällt die vorgenannte DEX Stablisierungsgebühr nicht an.",
```

Comment:
btw, it would make sense extract it to a const and just reference that here, so simpler refactor/change!

Reply:
works like:
tsx -> translate('components/ConversionBreakdown', 'From {{tokenA}} to {{tokenB}}', { tokenA: inToken, tokenB: outToken})
x.json -> "From {{tokenA}} to {{tokenB}}": "Von {{tokenA}} zu {{tokenB}}"

Reply:
Let's do this for the next change 👍